### ISSUE TITLE
Can't connect Specter to my external node

### ISSUE THREAD

**Issue by fatrattombala** [cryptoadvance/specter-desktop#872]
I’m not able to connect Specter Desktop to my own external Raspi-node. I’m a node-noob, following the instructions of Raspibolt (https://stadicus.github.io/RaspiBolt/). The node is up and running and currently syncing (90%). The ufw-firewall is allowing port 8332 for incoming RPC requests. I tried to set the bitcoin.conf (Raspibolt user "bitcoin") parameters **rpcallowip** and **rpcbind**, as it was suggested in the FAQ (https://github.com/cryptoadvance/specter-desktop/blob/master/docs/faq.md#trying-to-connect-specter-desktop-to-my-remote-node-on-my-lan-few-times-but-no-success-bitcoinconf-has-the-server1-option-should-there-be-something-else-since-i-get-this-error-process-finished-with-code--1error-message-failed-to-connect-message). "server=1" is set, as well as the rpcuser ("raspibolt") and rpcpassword (with some blanks, but also tried it without - no effect). For rpcallowip I tried i.e. "0.0.0.0" and "192.168.178.*" and "192.168.178.0/24", the same for rpcbind. In Specter Desktop, I disabled "Auto-detect", username and PW are the same as in the bitcoin.conf, host is "http://192.168.178.45", and the port 8332. Connection always fails with the output "Process finished with code -1Error message: Failed to connect!". 

Anybody an idea what I can do to connect to the node?

**Comment by fatrattombala** [comment]
finally found a solution after a lot of trial-and-error: **rpcallowip**=192.168.178.55/24 (IP-adress of my PC in my home-network) and **rpcbind**=192.168.178.45 (the IP of my Raspi with the node running on it; _without_ /24). 0.0.0.0 for the rpcbind also works, but is not recommended due to security reasons.

**Comment by k9ert** [comment]
Great you find a solution. Sorry for the late response. Can you check whether your specific issue could have been solved with this document:
https://github.com/cryptoadvance/specter-desktop/blob/master/docs/connect-your-node.md
... Or what we could improve in this document?

**Comment by fatrattombala** [comment]
Stupid me couldn't find this document, thanks for the link. It would have probably helped. I would add the following:

---
Example: Specter is running on your PC with the local network IP-address 192.168.178.55. You want to allow this machine to be able to send RPC requests and connect Specter to the external node, which has the IP-address 192.168.178.45 in your local network. You also want to be able to do RPC requests on the external node itself, i.e. directly or logged in with a ssh-session. You have to specify the two options in bitcoin.conf as follows:
```
rpcallowip=192.168.178.55/24
rpcbind=192.168.178.45
rpcbind=127.0.0.1
```
Note: both options can be specified multiple times.
---
In fact, I have proposed these changes in the document with a pull request.

**Comment by k9ert** [comment]
Great! Thank you very much. The stuff mentioned in your screenshot is addressed right now in #888. We're now using tool-tips and the link-color is grey. Much better hopefully!
Assuming the both PRs will get merged, i'm closing this here!
